### ISSUE TITLE
fs argv subdir mismatch

### ISSUE THREAD

**Issue by runlow** [nesbox/TIC-80#1635]
**Expected:**
When you run `tic80 --fs=./tic tic/music/song1.tic`, then make some changes, press ctrl+s - it should write to same file realpath as the one provided in the command

Alternatively - if the file provided as an argument **already is** in the "fs dir" - then it should respect the directory structure in "fs dir" when saving

**Actual:**
It will create a file tic/song1.tic which you have to manually move to the subdirectory "music" each time you save it.

This does not happen when you load the cart inside tic80, only if you open it from the system shell

**Solutions/Workarounds/Ideas:**

- tic80 should detect if the realpath of the file path provided in argv as a command parameter overlaps with the "fs dir", and if it does then respect the subdirectory in fs dir when saving
- `tic80 --fs=./tic --cmd='load music/song1.js & run'` - workaround, it works but awkward to use
- could write a shell script "wrapper" that uses the workaround and detects if an argument is a file, then detects if the file is in the "fs" dir

**Comment by nesbox** [comment]
Agreed, we have to load the cart relatively to the path provided in `--fs` arg.
Thank you.
